lib/proportions.js

```diff
@@ -64,9 +64,8 @@ define(['d3-interpolate', 'snabbdom', 'filters/genericnode', 'helper'],
           var th = V.h('th', a);
           var td = V.h('td', V.h('span', {
             style: {
-              width: Math.round(v * 100) + '%',
-              backgroundColor: scale(v),
-              color: 'white'
```

Comment:
Is there a reason, why you remove the font-color?
Maybe it should be store here: https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_proportion.scss#L17

Reply:
Is same color as body. Will be inherited.

Reply:
https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_base.scss#L4 no there it is black ^^

Reply:
yeah - you are right - but is css - i fix it

Reply:
added